src/Symfony/Component/Intl/Resources/bin/update-data.php

```diff
@@ -195,6 +196,7 @@
         $targetDir.'/'.Intl::LOCALE_DIR,
         $targetDir.'/'.Intl::REGION_DIR,
         $targetDir.'/'.Intl::SCRIPT_DIR,
+        $targetDir.'/'.Intl::ZONE_DIR,
```

Comment:
Is there a reason to call this `Intl::ZONE_DIR` instead of `Intl::TIMEZONE_DIR` and `ZoneDataGenerator` instead of `TimeZoneDataGenerator`, etc. ?

Reply:
hm,  not really i guess :) i started of using ICU terminology from https://github.com/unicode-org/icu/tree/master/icu4c/source/data/zone (`zone/`). Ill patch this :+1: